frontend/app/src/theme/light.ts

```diff
@@ -14,6 +14,7 @@ const disabledTxt = txtLight;
 
 export function lightTheme(): Theme {
     return {
+        hidden: false,
```

Comment:
This field is only set on signals and light. Is that intentional?

Reply:
Yes, dark inherits light and all the other themes inherit from one of those two.